.github/workflows/pep8.yaml

```
@@ -0,0 +1,39 @@
+name: PEP8

```

I think that name would be a little more precise.
```suggestion
name: PEP8 linter
```